### PR TITLE
[SREDEV-10627] Added openssh to the dockerfile

### PR DESCRIPTION
**Pull request by venuspathak7522**
Added the ability to install openssh through the image.

Background: ssh is required for authentication to github by gomplate command. The goal is to read whitelisted cidrs from a central location in github to different repos which is achieved using gomplate.

Ran ssh-agent in the container locally:
image

### CODE REVIEW

File: tiny-build/Dockerfile [hunk 1]
```diff
@@ -32,6 +32,8 @@ RUN apk add --no-cache \
     jq \
     curl
 
+RUN apk add --update --no-cache openssh
```
Comment (zaafar) [inline tiny-build/Dockerfile:35]: Why not just add it after `curl` (line 33). Do we require the --update flag?

Comment (venuspathak7522) [reply tiny-build/Dockerfile:35]: I added --update to make sure it always adds in the latest version

Comment (zaafar) [reply tiny-build/Dockerfile:35]: That's the default behavior of `apk add` command unless the application (i.e. `openssh`) is already installed.

**Review by zaafar — APPROVED** [review]

File: tiny-build/Dockerfile (hunk 1 above)
Comment (venuspathak7522) [reply tiny-build/Dockerfile]: Sure, done

**Review by zaafar — APPROVED** [review]

**Review by aravandi — APPROVED** [review]

**Review by abuokoene — APPROVED** [review]

**Review by jennliu — APPROVED** [review]

**Pull request merged**